package.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "@novuhq/bridge-sync-action",
+  "name": "@novuhq/novu-sync-action",
   "description": "GitHub Actions TypeScript template",
-  "version": "0.0.0",
+  "version": "0.0.4",
```

Comment:
@rifont I've aligned this with the version we have in the sync modal, let me know if it should be something else

Reply:
Let's publish this as version `2.0.0` to reflect the v2 release, so we'll update both here and the GH action version via GH releases